cfme/web_ui/paginator.py

```diff
@@ -16,6 +16,9 @@
 _page_cell = '//td//td[contains(., " of ")]|//li//span[contains(., " of ")]'
 _check_all = Input("masterToggle")
 
+_regexp = {version.LOWEST: r"(?:Item|Items)*\((\d+).*of (\d+)\)(?:item|items?)*",
+                    "5.5": r"Showing (\d+)-(?:\d+) of (\d+) (?:item|items?)?"}
```

Comment:
I wonder also if it makes sense......can you do group naming in python regexp? I thought you could.....could make it more readable later in the code?